project/AppDependencies.scala

```diff
@@ -3,31 +3,31 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  
+
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-26"     % "1.7.0",
-    "uk.gov.hmrc" %% "domain"                 % "5.8.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client"    % "9.0.0-play-26",
-    "uk.gov.hmrc" %% "tax-year"               % "1.0.0",
-    "uk.gov.hmrc" %% "play-partials"          % "6.10.0-play-26",
-    "uk.gov.hmrc" %% "emailaddress"           % "3.4.0",
-    "uk.gov.hmrc" %% "bulk-entity-streaming"  % "1.0.0",
-    "uk.gov.hmrc" %% "auth-client"            % "3.0.0-play-26",
-    "uk.gov.hmrc" %% "govuk-template"         % "5.54.0-play-26",
-    "uk.gov.hmrc" %% "play-ui"                % "8.9.0-play-26",
-    "com.typesafe.play" %% "play-json-joda"   % "2.7.4",
-    "com.typesafe.play" %% "play-iteratees"   % "2.6.1"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-26"  % "2.24.0",
+    "uk.gov.hmrc" %% "domain"                      % "5.9.0-play-26",
+    "uk.gov.hmrc" %% "http-caching-client"         % "9.1.0-play-26",
+    "uk.gov.hmrc" %% "tax-year"                    % "1.1.0",
+    "uk.gov.hmrc" %% "play-partials"               % "6.11.0-play-26",
+    "uk.gov.hmrc" %% "emailaddress"                % "3.5.0",
+    "uk.gov.hmrc" %% "bulk-entity-streaming"       % "1.0.0",
+    "uk.gov.hmrc" %% "auth-client"                 % "3.0.0-play-26",
+    "uk.gov.hmrc" %% "govuk-template"              % "5.55.0-play-26",
+    "uk.gov.hmrc" %% "play-ui"                     % "8.9.0-play-26",
```

Comment:
can you check if is a newer version for play-ui?

Reply:
upgrading to 8.11.0-play-26